roles/config-wireguard/config/ams.yml

```diff
@@ -322,4 +322,11 @@ wg_peers:
   remote: ams.dn42.join.yt:23002
   wg_pubkey: h/wiLWhZgRZUQAiWfpoV7va3hnvdJqeJK2ZoRgCcFkM=
   peer_v4: null
-  peer_v6: fe80::1409:201
\ No newline at end of file
+  peer_v6: fe80::1409:201
+
+- name: dn42-alemal
+  port: 21525
+  remote: uk1.alemal.se:51080
+  wg_pubkey: 1JwYl6O6Y94PiKe5dXcgwMYjDxRK3im8mqvbZIeMSDE=
+  peer_v4: None
```

Comment:
Amended in c7350566a75c4c787154bfb9885be667e7d64b9c: this needs to be `null` to not be interpreted as a string in YAML